packages/@aws-cdk/aws-cloudfront/lib/web_distribution.ts

```diff
@@ -344,6 +345,27 @@ export interface Behavior {
    */
   readonly maxTtlSeconds?: number;
 
+  /**
+   * Declares associated lambda@edge functions for this distribution behaviour.
+   *
+   * @default undefined
```

Comment:
`undefined` is obviously the default *value* if nothing is passed, but we're more interested in the default *behavior*.

Write something like this: `@default - No lambda functions are associated` (or whatever makes sense for this feature).

Reply:
Sure, I'll update that